src/sass/modules/_viewer.scss

```diff
@@ -3,6 +3,10 @@
 .openseadragon-canvas {
     outline: none !important;
     cursor: move;
+    height: calc(100% - 20px) !important;
+    width: calc(100% - 20px) !important;
+    top: 10px !important;
+    left: 10px !important;
```

Comment:
Making all these `!important` kinda stinks but it was the only way I could see us overriding styles applied directly on DOM by the OSD library. 
![image](https://user-images.githubusercontent.com/13315416/115300311-d1c6d080-a12d-11eb-80e7-87346a237a32.png)